.github/workflows/actions/auto-request-reviews/main.js

```diff
@@ -0,0 +1,45 @@
+const core = require('@actions/core');
```

Comment:
This file is just the compiled output from the action source code.

Reply:
is the idea that we want to cut down on startup time by not having to do a compilation step? if so, would it be too heavy to try to do some sort of content-addressed hash of the current state of the `main.ts` file? Just might be nice to avoid having to do a PR which just commits the new file if something changes.

Maybe just running the script with `ts-node` ?

Reply:
This is my first delve into custom Github actions, so what I say here might be completely wrong lol

AFAICT, there's no way to execute the uncompiled TS code with `ts-node` as a part of the workflow (which is a bummer cuz that would be handy). Also, most of the resources I have found walkthrough manually compiling the code and committing that to the repo. So, I feel like that's pretty standard.

Couple thoughts on the need to update this when the action changes:
1. I realistically don't see the action changing too much. Maybe early on as we're feeling things out, but in the long run it's not likely
2. In the cases where we would change the action code, we'd likely want to compile it locally anyway to make sure it builds successfully. So it's just a matter of committing it at that point. Also, if you want to use `act` to test it locally, you need to compiled code anyway.

Reply:
cool, that all makes sense - just figured I would ask because the big JS blob made me wonder a little bit haha